### ISSUE TITLE
Support Sprint Field for Issues

### ISSUE THREAD

**Issue by base698** [baohaojun/org-jira#18]

**Comment by tomjack** [comment]
I sort of solved this for myself, using the following function:

```
(defun add-to-sprint ()
  (interactive)
  (jiralib-update-issue
   (org-jira-get-from-org 'issue 'key)
   '(("customfield_10281" . "2689"))))
```

`(org-jira-get-from-org 'issue 'key)` here just gets the key for the issue at point.

You will certainly need to change 2689 to your sprint id, and I guess you probably need to change the 10281 as well.

I found the numbers this way:

- Find the link like `/issues/?jql=sprint = 2689` on the backlog, near the sprint name. This gives us the 2689.
- Look at M-: (jiralib-get-issue "BLAH-0") for an issue BLAH-0 already in the sprint. In the custom field values, one should have a string with the sprint id, which is the one we need.

**Comment by baohaojun** [comment]
Nice! It won't be easy to add your code into org-jira though, because it's not general -- not every one is using sprint id, for e.g., I have not used it yet.

It would be good if org-jira can have some hooks, then you can define your logic in the hooks, but alas, I have to find time to do it first, or maybe someone can help to add it...

**Comment by partition21** [comment]
Yes, the general problem seems tricky... I left the snippet above in case someone else using sprints finds this issue and can get by with the hack for now.

By the way, thanks for this excellent project! It has made JIRA usable for me. :)
